client.go

```diff
@@ -63,6 +63,9 @@ type Client struct {
 	// Handler called when calling `Close` to clean up any remaining
 	// resources managed by `Client`.
 	closeHandler ICloseHandler
+
+	// Preserve the remote file permissions, modification time and access time
```

Comment:
Might be confusing to keep that here since you will be able to create an SCP client with `preserve` turned off and still use the `FileInfo` function. Then you will get no information at all even though you might have expected it.